tests/mhd/alfvenwing_test/Io.par

```diff
@@ -0,0 +1,99 @@
+ &filelist
+        base_filename='alfvenwing_3D_'
+        convert= F                  !
+        autoconvert = T
+        convert_type='vtuBCCmpi'
+        saveprim=.true.
```

Comment:
We usually have two parfiles in every test directory, one for manual runs and one for the regression tests.
I think you need a second parfile here in which `typefilelog='regression_test'` is specified, which is passed in `test.make`. Ideally you can lower the resolution/runtime in the second parfile (cfr. comment below).
See for example this folder: https://github.com/amrvac/amrvac/tree/master/tests/mhd/Kelvin_Helmholtz_3D